puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_develop_pr_core.yaml

```
@@ -27,15 +27,14 @@
           values:
             - postgresql-integrations
             - postgresql
-            - sqlite3
       - axis:
           type: label-expression
           name: slave
           values:
             - fast
     execution-strategy:
       combination-filter: >
-        ((database == 'postgresql') && ((ruby == '2.5') || (ruby == '2.6'))) || ((database == 'sqlite3') && (ruby == '2.5')) || ((database == 'postgresql-integrations') && (ruby == '2.5'))
+        ((database == 'postgresql') && ((ruby == '2.5') || (ruby == '2.6'))) || ((database == 'postgresql-integrations') && (ruby == '2.5'))
```

Untested, but how about:
```suggestion
        (database != 'postgresql-integrations') || (ruby == '2.5')
```

---

that would work, but I expect we'll be adding 2.7 soon, and at least for me it's easier to understand a list of the combinations we test instead of trying to figure the boolean logic of a negation

---

I found the shorter version a lot easier to understand. However, if you want, there's also
```suggestion
        (database == 'postgresql') || ((database == 'postgresql-integrations') && (ruby == '2.5'))
```

---

works for me, updated, squash on merge please.